intro-HPC/ch_VNC.tex

```diff
@@ -0,0 +1,132 @@
+\chapter{Graphical applications with VNC}
+\label{ch:vnc}
+
+It's important to remember that VNC sessions are permanent. They survive network
```

Comment:
First explain what a VNC session *is*?

And this paragraph belongs below, after showing how to use `vncserver` to start a VNC server